recipe/meta.yaml

```diff
@@ -28,22 +29,24 @@ requirements:
     # that was built with openblas rather than mkl. Openblas typically pairs with llvm-openmp and mkl with intel-openmp on osx-64. 
     # By default, mkl variants are picked. By picking the variant that uses openblas rather 
     # than mkl, intel-openmp will not be pulled in and so you will not have llvm-openmp / intel-openmp conflict. 
-    - blas=*=openblas # [osx and x86_64]
+    - blas=*=openblas  # [osx and x86_64]
     - scikit-build-core >=0.4.4
     - python
     - pip
-    - llvm-openmp  # [osx]
-    - libgomp      # [linux]
+    - llvm-openmp      # [osx]
```

Comment:
llvm-openmp should be moved to build, has it has a strong run_exports.
See: https://github.com/AnacondaRecipes/openmp-feedstock/blob/master/recipe/meta.yaml#L32

Reply:
Fixed

Reply:
After removing `lbgomp`, `osx-arm64` failed:
`    ['  ERROR (lightgbm,lib/python3.9/site-packages/lightgbm/lib/lib_lightgbm.so): /private/var/folders/nz/j6p8yfhx1mv_0grj5xl4650h0000gp/T/abs_28g70qf1va/croot/lightgbm_1713967649472/_build_env/lib/libomp.dylib not found in packages, sysroot(s) nor the missing_dso_whitelist.\n.. is this binary repackaging?']`

Reply:
I verified the correct run deps are set. It looks fine.